### DISCUSSION TITLE
Help me dedicate time to write the Sozi Guide

### DISCUSSION THREAD

**Discussion by aumouvantsillage** [senshu/Sozi#584] [Announcements]
Since 2009, a lot of people have tried Sozi and found it useful. Amazing creations have been made with it, and I have received incredible support from all over the world.

Sozi is still a personal project that I develop mostly alone in my spare time. Over the years, while many users, including myself, were expecting new features and bug fixes, writing code has often felt like a better use of my time than writing documentation.
Today, I realize that the documentation for Sozi needs to be updated and reorganized. I usually feel that some questions on the community forum would be easier to answer if there was a real Sozi user guide available.

For this reason, I have decided to rewrite the documentation of Sozi in the form of an online book. I have also reserved a very nice domain name for it: http://sozi.guide
Like Sozi, the book will be published under a free license, and will be available on the web with no ads and no paywall.

However, writing the Sozi Guide in my spare time is not a viable option today. This is why [I am starting a fundraising campaign](https://gofund.me/434c742e) to help me take as many days off from my job as necessary, so that I can focus on writing the book, create new examples and tutorials.

If you prefer, you can still donate via these other services. There is no minimum amount. Even small donations are welcome.
* [Buy me a coffee](https://www.buymeacoffee.com/THtbNvnqE)
* [Paypal](https://www.paypal.me/guillaumesavaton)
* [Liberapay](https://liberapay.com/aumouvantsillage/donate)

Thanks for your support. Please feel free to share your ideas about what you expect from the Sozi Guide.

**Comment by Moini** [comment]
Hi Guillaume, do you have a Mastodon / Twitter / ... account where you've posted this? I can't donate, but I could retoot from the Inkscape account.

**Reply by aumouvantsillage** [reply]
Hi @Moini 

Thanks a lot. That would be great!
At the moment, I really need some help spreading the word about this campaign: though I have more that 300 followers on Twitter, I can only reach a few of them at a time, and I get even fewer retweets :-(

I have posted this announcement [on Twitter](https://twitter.com/umouvantsillage/status/1455234389298884625) and [on Mastodon](https://mamot.fr/web/statuses/107203350572048161).

**Reply by Moini** [reply]
Retooted, and asked the Facebook and Twitter persons to share on their platforms. Hope it helps!

**Reply by aumouvantsillage** [reply]
Thanks again for your help.

Tweeting from Inkscape's account allowed to reach people who are interested in vector graphics and who didn't know Sozi, which is great for the visibility of the project.
I don't expect a sudden increase in donations, but it is likely that a few new users will try Sozi, and if they find it useful, they will understand how the Sozi Guide could help them. :smile:

**Comment by penyuan** [comment]
Thank you @aumouvantsillage for all your hard work! 🙇🏼‍♀️ I've been wanting to use Sozi for a few years now, but *the lack of documentation thus far has made learning it incredibly difficult for me*. Every time I try to use Sozi, I just get confused and give up... If a comprehensive set of documentation could be developed, that will help me learn to use Sozi and promote it to other people. Therefore, I definitely would like to make a donation!

Question: It's amazing that the documentation you're writing will also be under a free license! (is it CC BY or CC BY-SA?) But will it be managed in a Git repository like Sozi so that after you write it, others can help maintain the documentation to keep it in sync with new Sozi features?

**Reply by aumouvantsillage** [reply]
Hi @penyuan. I'm glad that you are still willing to use and support Sozi after your frustrating experience.

> (is it CC BY or CC BY-SA?) But will it be managed in a Git repository

Yes. I have chosen to publish it under a CC-BY-SA License, and the source code has [its own GitHub project](https://github.com/sozi-projects/Sozi-guide).
You can read more about this in the chapter [About this book](http://sozi.guide/en/about-this-book.html).

**Reply by penyuan** [reply]
Splendid! Just donated. Please keep up the great work!

**Reply by penyuan** [reply]
BTW, I just looked at the GitHub repository for the book. Having a `LICENSE` file in the repository is still a good idea, CC BY-SA 4.0 is possible and makes the repository more machine readable.

**Reply by aumouvantsillage** [reply]
I agree. I have just added the license file to the repository.
As far as I can tell, the license chooser at creativecommons.org does not propose to download a plain text version.
I found it [via a link that was posted on their blog in 2014](https://creativecommons.org/2014/01/07/plaintext-versions-of-creative-commons-4-0-licenses/).

**Comment by bidinou** [comment]
Hi ! Is it still possible to make donations ? :)

**Reply by aumouvantsillage** [reply]
Hi. Technically yes, it is still possible but please don't donate.
I have stopped working on Sozi several years ago and I am planning to remove all donation links related to the project.
Sozi itself is in "maintenance" mode at the moment and I have no time to continue working on the Sozi guide.

**Reply by penyuan** [reply]
> Hi. Technically yes, it is still possible but please don't donate. I have stopped working on Sozi several years ago and I am planning to remove all donation links related to the project. Sozi itself is in "maintenance" mode at the moment and I have no time to continue working on the Sozi guide.

Really sorry to hear that, but I understand if you are too busy with other things. 

Is there anyone else who is taking over the "maintenance" mode of Sozi? And can we support them?

**Reply by aumouvantsillage** [reply]
As far as I know, nobody has volunteered to continue developing Sozi.

**Reply by penyuan** [reply]
> As far as I know, nobody has volunteered to continue developing Sozi.

Understood, thank you for responding, and many thanks for all the work you put into Sozi!!

Hoping one day someone can pick up the ball on this great project again. ❤️

**Reply by bidinou** [reply]
Hi ! Thanks so much for taking the time to reply.
I am also quite sad to see this project not being taken over. I felt it was quite unique and empowering.

Thanks so much for your great work ! <3
